elasticmagic/src/commonMain/kotlin/dev/evo/elasticmagic/aggs/Metric.kt

```diff
@@ -117,6 +118,43 @@ data class MaxAgg<T>(
     override fun clone() = copy()
 }
 
+
+data class PercentileAggResult(val values: Map<Double, Double>) : AggregationResult {
+    companion object {
+        operator fun invoke(obj: Deserializer.ObjectCtx): PercentileAggResult {
+            val values = obj.obj("values")
+            return PercentileAggResult(
+                values.toMap().filter { it.value is Double }.map { it.key.toDouble() to it.value as Double }.toMap()
+            )
+        }
+    }
+}
+
+@Suppress("MagicNumber")
+data class PercentilesAgg(
+    val field: FieldOperations<*>,
+    val percents: List<Double> = listOf(1.0, 5.0, 25.0, 50.0, 75.0, 95.0, 99.0),
```

Comment:
I don't think we should copy percentiles from Elasticsearch defaults. Just make it nullable.

Reply:
Made it